open-media-match/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -75,12 +77,13 @@ def bank_update(bank_name: str):
     return jsonify(bank.as_storage_iface_cls())
 
 
-@bp.route("/bank/<int:bank_id>", methods=["DELETE"])
-def bank_delete(bank_id: int):
+@bp.route("/bank/<bank_name>", methods=["DELETE"])
+@utils.abort_to_json
+def bank_delete(bank_name: str):
     # TODO - rewrite using persistence.get_storage()
-    bank = database.Bank.query.get(bank_id)
+    bank = database.Bank.query.filter_by(name=bank_name).one_or_none()
     if not bank:
-        return jsonify({"message": "bank not found"}), 404
+        abort(404, "Bank not found")
```

Comment:
blocking: Deleting a bank that doesn't exist should return success (it may be the result of a retry).

Reply:
updated code and tests